content/applications/finance/fiscal_localizations/uruguay.rst

```diff
@@ -0,0 +1,321 @@
+
+The localization requires an Uruware account, which enables users to generate electronic documents
+within Odoo.
+
+Glossary
+------------
+
+The following terms are used throughout the Uruguayan localization:
+
+- **DGI**: *Dirección General Impositiva* is the government organization that enforces pay of
+  taxes in Uruguay.
+- **EDI**: *Electronic Data Interchange* refers to the sending of electronic documents.
+- **Uruware**: is the third-party organization that facilitates the interchange of electronic
+  documents between companies and the Uruguayan government.
+
+Configuration
+=============
+
+Modules installation
+--------------------
+
+:ref:`Install <general/install>` the following modules to get all the features of the Uruguayan
+localization:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 25 50
+
+   * - Name
+     - Technical name
+     - Description
+   * - :guilabel:`Uruguay - Accounting`
+     - `l10n_uy`
+     - The default :doc:`fiscal localization package <../fiscal_localizations>`, adds accounting
+       characteristics for the Uruguayan localization, which represent the minimum configuration
+       required for a company to operate in Uruguay according to the guidelines set by the |DGI|.
+       The module's installation automatically loads: chart of accounts, taxes, documents types, and
+       tax support types.
+   * - :guilabel:`Uruguay Accounting EDI`
+     - `l10n_uy_edi`
+     - Includes all the technical and functional requirements to generate and validate
+       :doc:`Electronics Documents <../accounting/customer_invoices/electronic_invoicing>`, based on
+       the technical documentation published by the |DGI|. The authorized documents are: invoices,
+       credit notes and debit notes.
+
+.. note::
+   Odoo automatically installs the base module *Uruguay - Accounting* when a database is
+   installed with `Uruguay` selected as the country.
+
+Configure your company
+----------------------
+
+To configure your company information, go to the :menuselection:`Settings --> Companies: Update
+Info` and configure the following:
+
+- :guilabel:`Company Name`
+- :guilabel:`Address`:
+
+  - :guilabel:`Street`
+  - :guilabel:`City`
+  - :guilabel:`State`
+  - :guilabel:`ZIP`
+  - :guilabel:`Country`
+
+- :guilabel:`Tax ID`: enter the identification number for the selected taxpayer type.
+- :guilabel:`DGI Main Branch Code`:
+
+After configuring the company in the database settings, navigating to :menuselection:`Contacts` and
+search for your company to verify the following:
+
+- Company type is set to :guilabel:`Company`.
+- Verify the :guilabel:`Identification Number` :guilabel:`Type` is :guilabel:`RUT / RUC`.
+
+.. _l10n_uy/uruware-account:
+
+Set up Uruware account
+----------------------
+
+To generate electronic documents in Odoo, a Uruware account is necessary.
+
+To set up a Uruware account, first be sure to have a valid Odoo subscription. Then follow the steps
```

Comment:
```suggestion
To set up a Uruware account, first be sure to have a valid Odoo subscription and the data of the company correctly filled in. Then follow the steps
```

Not sure if format-wise my suggestion will work, you can rephrase it, but be sure to add that info, the company's information has to be complete.